spec/index.bs

```diff
@@ -6073,12 +6084,23 @@ GPUComputePipeline includes GPUPipelineBase;
 
 ### Creation ### {#compute-pipeline-creation}
 
+A {{GPURenderPipelineDescriptor}} describes the state of a compute [=pipeline=]. See
```

Comment:
nit: old text, but "state" is a little confusing, kind of sounds like it can change.
```suggestion
A {{GPURenderPipelineDescriptor}} describes a compute [=pipeline=]. See
```